core/compose/compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.7'
+services:
+  blueos-core:
+    image: bluerobotics/blueos-core:1.2.0-beta.6
+    container_name: blueos-core
+    privileged: true
+    # This can be used as an alternative to ports binding
+    # Using host as a network mode is only supported for linux
+    # Ref: https://docs.docker.com/network/drivers/host
+    # network_mode: host
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /run/udev:/run/udev
+      - /etc/resolv.conf:/etc/resolv.conf
+      - /etc/machine-id:/etc/machine-id
+      - ./workspace/userdata:/usr/blueos/userdata
+      - ./workspace/logs:/var/logs/blueos
+      - ./workspace/config:/root/.config
+      - ./workspace/blueos:/etc/blueos
+      # Allow changes on repository reflects directly on the running container
+      - ../start-blueos-core:/usr/bin/start-blueos-core
+      - ../services:/home/pi/services
+    pid: "host"
+    environment:
+      - BLUEOS_DISABLE_SERVICES=cable_guy,wifi,commander
+      - BLUEOS_DISABLE_MEMORY_LIMIT=true
+    ports:
+      - "80:80"
+      - "81:81"       # Helper
+      - "2748:2748"   # NMEA Injector
+      - "5173:5173"   # Cockpit development port
+      - "6020:6020"   # MAVLink Camera Manager REST API and interface
+      - "6021:6021"   # MAVLink Camera Manager Stun and Signalling server
```

Comment:
```suggestion
      - "6021:6021"   # MAVLink Camera Manager Signalling server
```

Reply:
Can you explain why ?
Because the code says otherwise..

![image](https://github.com/bluerobotics/BlueOS/assets/1215497/05cbfc6f-3e4e-4278-b0bd-6545033fee37)

Reply:
https://github.com/mavlink/mavlink-camera-manager/issues/348